elyra/templates/components/generic_properties_template.jinja2

```diff
@@ -41,18 +41,34 @@
           "description": "For CPU-intensive workloads, you can choose more than 1 CPU (e.g. 1.5).",
           "minimum": 0
         },
+        "memory": {
+          "type": "integer",
+          "title": "RAM(GB)",
+          "description": "The total amount of RAM specified.",
+          "minimum": 0
+        },
         "gpu": {
           "type": "integer",
           "title": "GPU",
           "description": "For GPU-intensive workloads, you can choose more than 1 GPU. Must be an integer.",
           "minimum": 0
         },
-        "memory": {
+        "gpu_vendor": {
+          "type": "string",
+          "title": "GPU Vendor",
+          "description": "GPU Vendor, or K8s GPU resource type, default nvidia.com/gpu."
+        },
+        "gpu_memory": {
           "type": "integer",
-          "title": "RAM(GB)",
-          "description": "The total amount of RAM specified.",
+          "title": "GPU Mem",
+          "description": "Resource amount of GPU Memory. Must be an integer.",
           "minimum": 0
         },
+        "gpu_memory_vendor": {
```

Comment:
is the `gpu_memory_vendor` param exclusive to virtualized GPU resources? If so, can we change the name to something like `vgpu.....`